vignettes/email-overview.Rmd

```diff
@@ -0,0 +1,141 @@
+verschickt und beinhaltet folgende Ergebnisse:
+
+### Indexierungsstatus in Crossref
+
+Der Abschnitt fasst zusammen, für welche DOIs keine Metadaten in
+Crossref verfügbar sind. Es kann vielfältige Ursachen haben, warum eine DOI
+nicht in Crossref indexiert ist:
+
+-   DOI ist syntaktisch fehlerhaft
+
+-   DOI ist vom Verlag bereits registriert, der Artikel ist jedoch noch
+    nicht publiziert
+
+-   DOI wurde nicht bei Crossref, sondern bei einer anderen Agentur
+    registriert
+
+### Überblick: Crossref-Metadaten
+
+Der Abschnitt zeigt einen tabellarischen Überblick der Testergebnisse.
+Es werden absolute und relative Häufigkeiten dargestellt.
+
+Untersucht wurden:
+
+-   Verfügbarkeit von Creative-Commons-Lizenzen (CC License)
+
+-   Verfügbarkeit von regelkonformen CC-Lizenzen (Compliant CC)
+
+-   Unterstützung von Versionen für das Text- und Data-Mining (TDM) (TDM
+    Support)
+
+-   Vorhandensein von Förderinformationen (Funder info)
+
+-   Verknüpfung zwischen Publikationen und ORCID
+
+-   Unterstützung weitere offener Metadaten 
+    ([Open Abstracts](https://i4oa.org/) 
+    und [Open Citations](https://i4oc.org/))
+
+### Überblick Lizenzbedingungen
+
+Offene Lizenzen sind wesentlich, um den Zugang und die Nachnutzung von
+Open-Access-Artikeln unter bestimmten Bedingungen zu regeln. Einschlägig
+sind die Creative-Commons-Lizenzen (CC-Lizenzen) (siehe Überblick des
+[Bundesministeriums für Bildung und
+Forschung](https://www.bildung-forschung.digital/de/lizenzvertrag-und-offene-lizenzen-2650.html)).
+
+Die tabellarische Übersicht zeigt die Häufigkeit von CC-Varianten an. 
+CC-BY ist sowohl in den DEAL-Verträgen als auch beim DFG-Programm
+Open-Access-Publikationskosten der präferierte Lizenztyp.
+`NA` bedeutet, dass keine Lizeninformation in den Crossref-Metadaten
+verfügbar ist.
+
+### Ergebnis Compliance Check
+
+Die Tabelle zeigt die Anzahl und den Anteil **regelkonformer
+CC-Lizenzinformationen** laut dem HOAD-Algorithmus. Eine
+Lizenzinformation ist genau dann regelkonform, wenn
+
+- es sich um eine Creative-Commons-Standardlizenz handelt,
+
+- die CC-Lizenz sich auf die Verlagsversion, die sogenannte
+"version-of-record" bezieht,
+
+- und die CC-Lizenz mit Erscheinen ohne zeitlichen Verzug gültig ist
+(Ausschluss von delayed OA)
+
+Wir haben festgestellt, dass Verlage zwar immer häufiger CC-Lizenzen in
+den Crossref-Metadaten verzeichnen, einige allerdings die Lizenzinformation  
+nur unvollständig beschreiben. Dies kann Auswirkungen auf die automatisierte
+Auffindbarkeit und Datenanalysen von OA-Artikeln haben. 
+
+**Bei Problemen
+empfehlen wir daher, mit dem Verlag in Kontakt zu treten.**
+
+Mehr Hintergrundinformationen zur Überprüfung von Lizenzinformationen in
+Crossref im Kontext von OA-Transformationsverträgen:
+
+Michaela Voigt, "DEAL Open-Access-Option optimal nutzen --- ein
+Bibliothekspraxisbericht". LIBREAS. Library Ideas, 38 (2020).
+<https://libreas.eu/ausgabe38/voigt/>
```

Comment:
see https://subugoe.github.io/muggle/articles/communicating.html for citation on our websites.

will fix in commit